From 7cf5d9d4ea948d2b93b44f592087139c05f0149a Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Tue, 22 Sep 2020 12:13:40 +0200
Subject: [PATCH] Disable usb_xhci

Remove usb xhci configs to correct error
Unexpected XHCI event TRB, skipping... (bd8bda70 00000000 13000000 07008400)
BUG at drivers/usb/host/xhci-ring.c:496/abo!
BUG!
resetting ...

Upstream-status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io>
---
 configs/cl-som-imx8_defconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/cl-som-imx8_defconfig b/configs/cl-som-imx8_defconfig
index 99620b0..b9291e2 100644
--- a/configs/cl-som-imx8_defconfig
+++ b/configs/cl-som-imx8_defconfig
@@ -57,9 +57,9 @@ CONFIG_DM_REGULATOR_GPIO=y
 CONFIG_DM_THERMAL=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
-CONFIG_USB_XHCI_HCD=y
-CONFIG_USB_XHCI_DWC3=y
-CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_XHCI_HCD=n
+CONFIG_USB_XHCI_DWC3=n
+CONFIG_USB_EHCI_HCD=n
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC3_GADGET=y
 CONFIG_USB_GADGET=y
-- 
2.7.4

